telegram/_message.py

```diff
@@ -1572,23 +1683,26 @@ async def reply_text(
         For the documentation of the arguments, please see :meth:`telegram.Bot.send_message`.
 
         Keyword Args:
-            quote (:obj:`bool`, optional): If set to :obj:`True`, the message is sent as an actual
-                reply to this message. If ``reply_to_message_id`` is passed, this parameter will be
-                ignored. Default: :obj:`True` in group chats and :obj:`False` in private chats.
+            quote (:obj:`bool`, optional): |reply_quote|
```

Comment:
please add deprecation notice, both as warning (probably in `_parse_quote_argumets`) and as docstring